frontend/public/components/utils/units.js

```diff
@@ -101,19 +101,12 @@ const convertValueWithUnitsToBaseValue = (value, unitArray, divisor) => {
   return { value, unit };
 };
 
-const round = units.round = (value) => {
+const round = units.round = (value, decimalPlace) => {
```

Comment:
If we're reworking this, we should consider `Intl.NumberFormat` to format correctly for the user's locale:

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/NumberFormat

We use it for the overview:

https://github.com/spadgett/console/blob/a69288f379eb9817b2fc798c977ad28d17a6a3a5/frontend/public/components/overview/project-overview.tsx#L27

It has handling for formatting as percent as well, which I see you added below.